packages/retail-ui-extensions/src/components/Stepper/Stepper.ts

```diff
@@ -0,0 +1,12 @@
+import {createRemoteComponent} from '@remote-ui/core';
+
+export interface StepperProps {
+  initialValue: number;
+  onValueChanged: (value: number) => void;
+  useDefaultMargin?: boolean;
+  isDisabled?: boolean;
```

Comment:
Total nitpick, isn't the prop typically called just `disabled`? https://www.w3schools.com/jsref/prop_pushbutton_disabled.asp

Reply:
You are absolutley right, I like `disabled` more. You might see a lot of is `isSomething` in POS code base though